src/MessagePack/MessagePackSerializer.Json.cs

```diff
@@ -390,8 +390,7 @@ private static void ToJsonCore(ref MessagePackReader reader, TextWriter writer,
                         writer.Write(dt.ToString("o", CultureInfo.InvariantCulture));
                         writer.Write("\"");
                     }
-#if !UNITY_2018_3_OR_NEWER
-                    else if (extHeader.TypeCode == ThisLibraryExtensionTypeCodes.TypelessFormatter)
+                    else if (extHeader.TypeCode == ThisLibraryExtensionTypeCodes.TypelessFormatter && !MessagePackUnityOptions.Unity2018_3orNewer)
```

Comment:
This is the _only_ use of the `Unity2018_3orNewer` property. Do we really need it? If not, then we don't need the new `MessagePackUnityOptions` class at all.

Reply:
https://github.com/MessagePack-CSharp/MessagePack-CSharp/commit/9dd83054c521ad7a2e561edb376b5f97a397a3f2#diff-aa1ab988b6f1fc9eb25bc3793817ec1963bb46aa583edc8cfdaef66e0f09d23c

Originally `!UNITY_2018_3_OR_NEWER` was `NETSTANDARD1_4`.
And the reason why `#if` was inserted was removing compile errors.
I think that there seems no problem to remove `Unity2018_rorNewer`.